tests/test_utils.py

```diff
@@ -273,11 +422,41 @@ def test_fuzz_sub_pixel_read(_source_image):
         ow = random.randint(4, 128)
         oh = random.randint(4, 128)
         output = utils.image.sub_pixel_read(
-            test_image, bounds, (ow, oh), interpolation="linear"
+            test_image,
+            bounds,
+            (ow, oh),
+            interpolation="linear",
         )
         assert (ow, oh) == tuple(output.shape[:2][::-1])
 
 
+def test_fuzz_padded_sub_pixel_read():
+    """Fuzz test for numpy sub-pixel image reads with padding."""
+    random.seed(0)
+
+    image_path = Path(__file__).parent / "data" / "source_image.png"
```

Comment:
Please use `_source_image`

Reply:
Fixed in this commit https://github.com/TIA-Lab/tiatoolbox/pull/61/commits/8b687c59ac3e9b5d2fc296ded0cae17e148670d3